go.mod

```diff
@@ -88,7 +86,7 @@ require (
 	github.com/sirupsen/logrus v1.4.2
 	github.com/stretchr/testify v1.4.0
 	github.com/tomnomnom/linkheader v0.0.0-20180905144013-02ca5825eb80
-	github.com/urfave/cli v1.21.0
```

Comment:
👀  Might need to pin this version if its a used by rancher machine

Reply:
its the rancher dependency. this just gets it current to what is in master already.